conans/test/functional/toolchains/meson/test_ios.py

```diff
@@ -71,11 +71,13 @@ def env(self):
         cflags += " -isysroot " + self.xcrun.sdk_path
         cflags += " -arch " + to_apple_arch(self.arch)
         cxxflags = cflags
+        ldflags = cflags
```

Comment:
This looks wrong on the surface.

Reply:
according to @SSE4 apparently this is correct

Reply:
anyway, changed the code to be more accurate